plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -112,9 +114,40 @@ describe('PersonState.update()', () => {
         )
     })
 
+    it('updates person properties if creation in parallel was slow', async () => {
```

Comment:
This is kind of a poor test IMO.

Reasoning:
- The test name and what's going on in the test doesn't compute (what does "creation in parallel mean")
- For a unit test, this isn't consistently showing a case, but there's an implicit race condition going on there.
- I think that this unit tests for this fix would not have failed before this change.

Note that this should be a unit test. If you have a e2e test in mind, add that elsewhere.

How you could hypothetically test this:
1. add a `spyOn` and make something throw the first time it's called, name the test about retries
2. demonstrating race conditions: add a `spyOn` onto the critical area, make it await a custom promise you create at top level. Wait until code reaches that point, call/await the racing code, resolve and check results.

Reply:
> I think that this unit tests for this fix would not have failed before this change.

It did fail before, but good points about testing the race condition in a better way